app/build.gradle

```diff
@@ -26,7 +26,7 @@ task conanInstall {
     ["armv7", "armv8", "x86", "x86_64"].each { String arch ->
         exec {
             commandLine(
-                "conan", "install", ".",
+                "/home/tom/Documents/ODR/OpenDocument.droid/bin/conan", "install", ".",
```

Comment:
Not sure if we should keep it like this, but at least that makes it obvious what needs to be changed in order to make it work on another system?

Reply:
I fear that will break the CI